src/ServerRequest.php

```diff
@@ -12,11 +12,8 @@ class ServerRequest implements ServerRequestInterface
 {
     use ServerRequest\GlobalEnvironment;
     use ServerRequest\ServerParams;
-    use Message\ProtocolVersion;
```

Comment:
`Response` should only use its own traits. Message traits should just serve as base.